packages/package_velvet_1_2_10/tool_dependencies.xml

```diff
@@ -0,0 +1,31 @@
+<tool_dependency>
+    <package name="velvet" version="1.2.10">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://github.com/dzerbino/velvet/archive/9adf09f7ded7fedaf6b0e5e4edf9f46602e263d3.zip</action>
```

Comment:
Then I'd prefer this to be changed to d430f1ade319af582dc7bb6886ea904abd6337bd.zip or change the package version to the commit ID.